packages/flutter_background_service/pubspec.yaml

```diff
@@ -15,7 +15,11 @@ dependencies:
     sdk: flutter
 
   flutter_background_service_platform_interface: ^5.0.0
-  flutter_background_service_android: ^6.0.1
+  flutter_background_service_android:
+    git:
+      url: https://github.com/ekasetiawans/flutter_background_service.git
+      ref: 37200c0086ffd0284ffb18b033ef90e499536d28
+      path: packages/flutter_background_service_android
```

Comment:
latest version for flutter_background_service_android is v5.0.2.
What is the reason of not using that?
In other words, why use a specific git revision => '37200c0086ffd0284ffb18b033ef90e499536d28?